src/components/LenderProfile/LenderLoansList.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+	<section v-if="lenderLoans.length > 0" class="tw-my-8">
+		<h4 class="data-hj-suppress tw-mb-1">
+			{{ lenderLoansTitle }}
+		</h4>
+		<p class="tw-mb-2">
+			{{ showedLoans }}
+		</p>
+
+		<div class="tw-grid tw-grid-cols-1 md:tw-grid-cols-2 lg:tw-grid-cols-4 tw-gap-3">
+			<new-home-page-loan-card
+				class="!tw-max-w-full"
+				v-for="(loan, index) in lenderLoans"
+				:item-index="index"
+				:key="`loan-${loan.id}`"
+				:loan-id="loan.id"
+			/>
+		</div>
+		<kv-pagination
+			class="tw-mt-4"
+			v-if="totalCount > loansLimit"
+			:limit="loansLimit"
+			:total="totalCount"
+			:offset="loansOffset"
+			@page-changed="pageChange"
+		/>
+	</section>
+</template>
+
+<script>
+import _isEqual from 'lodash/isEqual';
+import _get from 'lodash/get';
+import numeral from 'numeral';
+import logReadQueryError from '@/util/logReadQueryError';
+import lenderLoansQuery from '@/graphql/query/lenderLoans.graphql';
+import NewHomePageLoanCard from '@/components/LoanCards/NewHomePageLoanCard';
+import KvPagination from '~/@kiva/kv-components/vue/KvPagination';
+
+export default {
+	name: 'LenderLoansList',
+	inject: ['apollo', 'cookieStore'],
+	components: {
+		NewHomePageLoanCard,
+		KvPagination,
+	},
+	props: {
+		publicId: {
+			type: String,
+			required: true,
+		},
+		lenderInfo: {
+			type: Object,
+			required: true,
+		},
+	},
+	data() {
+		return {
+			lenderLoans: [],
+			loansLimit: 12,
+			loansOffset: 0,
+			totalCount: 0,
+			pageQuery: { loans: '1' },
+		};
+	},
+	computed: {
+		lenderLoansTitle() {
+			return this.lenderInfo?.name
+				? `${this.lenderInfo.name}'s loans`
+				: 'Loans';
+		},
+		showedLoans() {
+			return this.totalCount > 1
+				? `${this.totalCount} loans`
+				: `${this.totalCount} loan`;
+		},
+		urlParams() {
+			const loansPage = Math.floor(this.loansOffset / this.loansLimit) + 1;
+
+			return { loans: loansPage > 1 ? String(loansPage) : undefined };
```

Comment:
Should this query param be `page`?

Reply:
As @emuvente said:

> We will need to paginate for teams. Possibly invitees and dedications too

So I think is good using loans.